tests/storage/concurrent_vm_boot/conftest.py

```
@@ -0,0 +1,124 @@
+                    failed_indices.append(vm_index)
+
+        if failed_indices:
+            pytest.fail(f"{len(failed_indices)}/{NUM_CONCURRENT_VMS} VMs failed to create: {failed_indices}")
+
+        yield vms
+    finally:
+        cleanup_errors = []
+        for vm in vms:
+            try:
+                vm.clean_up()
+            except Exception as error:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)